pkgs/top-level/python-packages.nix

```diff
@@ -6990,6 +6990,28 @@ in {
     };
   };
 
+  ezdxf = buildPythonPackage rec {
```

Comment:
Is there a clear criterion for what belongs inline in python-packages.nix and what belongs in a separate file? I just looked and guessed.